### ISSUE TITLE
Not working in mac M1

### ISSUE THREAD

**Issue by andrecavallari** [manusa/electronim#329]
I've downloaded the version 0.0.95 from the releases page, the [.dmg for the arm64](https://github.com/manusa/electronim/releases/download/v0.0.95/electronim-mac-arm64.dmg), installed but when I try to execute it says that `ElectronIM is damaged and can't be open, you can move it to trash`

MacOS 13.5.2 (Ventura)
Macbook pro M1 max 32gb

**Comment by manusa** [comment]
Hi @andrecavallari 

Thanks for reporting the issue.

Unfortunately I don't own an Apple and it's impossible for me to try and reproduce the issue.

The only thing that changed in the latter versions (in your case 0.0.95) was the bump of Electron Builder.

Is this problem also reproducible if you download version [0.0.93](https://github.com/manusa/electronim/releases/download/v0.0.93/electronim-mac-arm64.dmg)?

Is there any way to get more details from your failed execution?

**Comment by lepatrick** [comment]
Yes same Problem with 0.0.93. @andrecavallari did you also run MacOS 14.0?

**Comment by andrecavallari** [comment]
@lepatrick , I did in 13, but now same in 14...

@manusa , This message is all I have here :(

**Comment by andrecavallari** [comment]
![image](https://github.com/manusa/electronim/assets/1050485/c4518fdf-d4d8-4a91-95f7-90dea3d8bdfb)

I still have this message, which means: "ElectronIM is damaged and can't be opened. You must move to Trash"

Latest version, MacOS 14

**Comment by manusa** [comment]
https://github.com/electron-userland/electron-builder/issues/5850

**Comment by opusonline** [comment]
Here's a quick workaround that worked for me:

Download the mac-arm64 version and install the ElectronIM.app into your Applications folder. Open terminal and navigate to the Applications folder and then type:
```
xattr -cr ElectronIM.app
```
This removes the "provenance" and "quarantine" extended attributes.

**Comment by opusonline** [comment]
I recently came across this issue and found the following solution for distributing my own created app.

"Ad-hoc signing"
https://eclecticlight.co/2019/01/17/code-signing-for-the-concerned-3-signing-an-app/#:~:text=about%20signing%20it%3F-,Ad%20hoc%20signing,-For%20all%20my

This signs the app without a certificate. It circumvents the "damaged" blocking popup and instead just hints that the developer is unidentified.

**Comment by kyds3k** [comment]
This problem still exists . . . I saw an update was available and now I get this error message on my M2 Max MacBook Pro.

**Comment by manusa** [comment]
> Here's a quick workaround that worked for me:
> 
> Download the mac-arm64 version and install the ElectronIM.app into your Applications folder. Open terminal and navigate to the Applications folder and then type:
> 
> ```
> xattr -cr ElectronIM.app
> ```
> 
> This removes the "provenance" and "quarantine" extended attributes.

This is working for M4 too.

Link to other thread: [Mar 19, 2022 05:32 AM in response to MrHoffman (discussions.apple.com)](https://discussions.apple.com/thread/253714860?answerId=256963522022&sortBy=best#256963522022)

This would be automatically fixed in case the application was signed.
However, this has a [yearly cost of $99](https://developer.apple.com/support/compare-memberships/) which I'm not willing to pay since I don't like endorsing this kind of gatekeeping, monopolistic, and closed practices that go against the Free Open Source Software principles from organizations such as Apple.
If someone is willing to share their developer credentials to notarize/sign the application or willing to donate the $99/year fee the issue can be fixed without any other workaround.
